doc/api/packages.md

```diff
@@ -156,6 +156,20 @@ For completeness there is also `--input-type=commonjs`, for explicitly running
 string input as CommonJS. This is the default behavior if `--input-type` is
 unspecified.
 
+## Determining package manager
+
+> Stability: 1 - Experimental
+
+While all Node.js projects are expected to be installable by all package
+managers once published, their development teams are often required to use one
+specific package manager. To make this process easier, Node.js ships with a
+tool called [Corepack][] that aims to make all package managers transparently
+available in your environment - provided you have Node.js installed.
+
+By default Corepack won't enforce any specific package manager and will use
+generic "Last Known Good" versions, but you can improve this experience by
```

Comment:
How does Corepack determine which is the LKG version?

Reply:
By default it's checked-in within [this file](https://github.com/nodejs/corepack/blob/main/config.json), shipped with each Corepack release (so one Node release will always have the same LKG versions). It can be overridden by the user via [`corepack prepare --activate`](https://github.com/nodejs/corepack#corepack-prepare--nameversion) (but perhaps the `--activate` flag should be renamed to something clearer 🤔).